msteams-platform/messaging-extensions/design/messaging-extension-design.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Designing your message extension
-description: Learn how to design a Teams message extension and get the Microsoft Teams UI Kit. Describes teams design guidelines reference message extensions tips best practice
+description: Learn how to design a Teams message extension and get the Microsoft Teams UI Kit. Describes about Teams design guidelines reference message extensions tips and best practice.
```

Comment:
No commas?

Reply:
Edited and added the commas in the description.